vulture/core.py

```diff
@@ -46,6 +47,34 @@
 if sys.version_info < (3, 4):
     IGNORED_VARIABLE_NAMES |= {"True", "False"}
 
+ISSUE_CODE_MAP = {
+    "attribute": "V001",
+    "class": "V002",
+    "function": "V003",
+    "import": "V004",
+    "property": "V005",
+    "variable": "V006",
+    "unreachable_code": "V007",
+}
+
+NOQA_REGEXP = re.compile(
+    # Use the same regex as flake8 does.
+    # https://gitlab.com/pycqa/flake8/-/tree/master/src/flake8/defaults.py
+    # We're looking for items that look like this:
+    # `# noqa: E123`
+    # `# noqa: E123,W451,F921`
+    # `# noqa:E123,W451,F921`
+    # `# NoQA: E123,W451,F921`
+    # `# NOQA: E123,W451,F921`
+    # `# NOQA:E123,W451,F921`
+    # We do not want to capture the ``: `` that follows ``noqa``
+    # We do not care about the casing of ``noqa``
+    # We want a comma-separated list of errors
+    # https://regex101.com/r/4XUuax/2 full explanation of the regex
```

Comment:
No need to copy the whole comment. The first three lines and a well-formed example are enough.